app/assets/javascripts/client/controllers/responses/ResponseCtrl.js

```diff
@@ -1,14 +1,38 @@
-PDRClient.controller('ResponseCtrl', [
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('ResponseCtrl', ResponseCtrl);
+
+  ResponseCtrl.$inject = [
     '$scope',
     '$timeout',
-    'SessionService',
-    'Assessment',
     '$stateParams',
-    function($scope, $timeout, SessionService, Assessment, $stateParams) {
-      $scope.user = SessionService.getCurrentUser();
-
-      $scope.assessmentId = $stateParams.assessment_id;
-      $scope.responseId   = $stateParams.response_id;
-      $scope.assessment   = Assessment.get({id: $scope.assessmentId});
-    }
-]);
+    '$modal',
+    'Assessment',
+    'LearningQuestion'
+  ];
+
+  function ResponseCtrl($scope, $timeout, $stateParams, $modal, Assessment, LearningQuestion) {
+
+    $scope.assessmentId = $stateParams.assessment_id;
+    $scope.responseId = $stateParams.response_id;
+    $scope.assessment = Assessment.get({id: $stateParams.assessment_id});
+
+
+    $scope.$on('responses-loaded', function() {
+      $timeout(function() {
+        LearningQuestion.exists({assessment_id: $stateParams.assessment_id})
+            .$promise
+            .then(function() {
+              // No-op - implies that a learning question exists for this user on this assessment
+            }, function() {
```

Comment:
I originally intended that endpoint to be a HEAD request, but our current version of Rails and HEAD requests don't seem to mix all that well.  In the context of the head, all I'm checking for is if it exists or not; if it doesn't then we should be opening the modal.

I agree here though that we should probably ensure that we're not getting a 500 back from the promise which would cause the modal to open.  A quick smoke test of this would be to attempt to load the page, and while it loads, turn off the server to see 500s come back.  The modal ideally shouldn't open if we get any 500s back.